pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -350,6 +354,52 @@ func (gce *GCECloud) ensureExternalLoadBalancerDeleted(clusterName, clusterID st
 	return nil
 }
 
+func (gce *GCECloud) deleteHealthCheckAndFirewall(service *v1.Service, name, clusterID string, hcName string) error {
```

Comment:
Is this function just moved up here with no significant change?

Reply:
No change. Just code reuse.